utils/outputwriter/outputcontent.go

```diff
@@ -335,7 +338,30 @@ func ApplicableCveReviewContent(severity, finding, fullDetails, cve, cveDetails,
 }
 
 func getJasDescriptionTable(severity, finding string, writer OutputWriter) string {
-	return NewMarkdownTable("Severity", "Finding").AddRow(writer.FormattedSeverity(severity, "Applicable"), finding).Build()
+	return NewMarkdownTable("Severity", "Finding").AddRow(writer.FormattedSeverity(severity, jasutils.Applicable.String()), finding).Build()
+}
+
+func getSecretsDescriptionTable(severity, finding, status string, writer OutputWriter) string {
+	columns := []string{"Severity", "Finding"}
+	applicability := jasutils.Applicable.String()
```

Comment:
We don't have `undetermined` for Secrets' Applicability? only `Applicable ` and `NotApplicable `?

Reply:
we have a diff statuses for it, I use the Applicable for the image that will be shown.
If `Inactive` I will show the severity image like `Not Applicable`, all other statuses are shown with severity image of `applicable`